### PR TITLE
*: Assorted internal-reference cleanups

### PR DESCRIPTION
**Pull request by wking**
This is mostly replacing `https://github.com/kubernetes/community/blob/master/...` links with relative links, because relative links are easier to maintain.  For example, if you rename `a.md` to `b.md` in a feature branch, updating an `../a.md` to `../b.md` target will work immediately (e.g. during pull-request review), while updating `https://github.com/kubernetes/community/blob/master/a.md` to `https://github.com/kubernetes/community/blob/master/b.md` will only work after your branch lands the rename in master.

I'm not sure why the `#L1454` fragment was part of the old sig-governance template sigs.yaml link, but it's been there since the template landed in 68658bff (#1830).  I've removed it in this commit.

The Kubernetes Charter README link in the sig-governance template could just be:

    [Kubernetes Charter README]: README.md

but I've sent it down to the project root and back to make life easier on new SIGs which copy/paste the file to a new location in this repository.  With this pull request they'll just need to drop one level of `../` to get the references working from places like `sig-contributor-experience/charter.md`.

I've also chased a few files and anchors as they've moved around, e.g. the `OBSOLETE_templates.md` rename from 18958f82 (#1010) and the "and maintenance" header addition from f471f6b6 (#1215).

This pull request isn't complete, but it's a step in the right direction and all I had time for tonight ;).

### CODE REVIEW

**Comment by cblecker** [comment]
Please have a look at https://github.com/kubernetes/community/blob/master/sig-contributor-experience/markdown-link-style-guide.md. There are a lot of positional references (e.g. `../sigs.yaml` that can just be `/sigs.yaml` instead.

**Comment by wking** [comment]
> Please have a look at https://github.com/kubernetes/community/blob/master/sig-contributor-experience/markdown-link-style-guide.md...

Thanks for the policy link.  It's probably worth dropping a reference to it in the root `CONTRIBUTING.md`, but I'll leave that for future work.

I've pushed 7d863216 -> e008e0de replacing all of my `../` and also fixing a stale `0000-kep-template.md` link in `committee-steering/governance/sig-governance-template-short.md` that hadn't been updated in #1531.

**Comment by wking** [comment]
Rebased onto master with e008e0de -> 13aae20a.  Git reported no conflicts, but comparing the diffs I think GitHub's issue was the rename in #2246.  Is there anything I can do to help this PR along so I can avoid future rebasing?  I'm happy to break it down into a bunch of per-file PRs or whatever if that would help with review.

**Comment by cblecker** [comment]
As you've modified a template, one of the docs needs to be regenerated.

**Comment by wking** [comment]
> As you've modified a template, one of the docs needs to be regenerated.

Ah, thanks.  Done with 13aae20a -> ca74dc32.

File: generator/list.tmpl [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-time bounded Working Groups, and the [community meeting](communication.md#weekly-meeting).
+time bounded Working Groups, and the [community meeting](communication/README.md#weekly-meeting).
 
 SIGs follow these [guidelines](governance.md) although each of these groups may operate a little differently
 depending on their needs and workflow.
 
 Each group's material is in its subdirectory in this project.
 
-When the need arises, a [new SIG can be created](sig-creation-procedure.md)
+When the need arises, a [new SIG can be created](sig-governance.md#sig-creation-and-maintenance-procedure.md).
```
Comment (cblecker) [inline generator/list.tmpl]: This anchor looks bad.

Comment (wking) [reply generator/list.tmpl]: > This anchor looks bad.

Good catch.  Fixed with ca74dc32 -> 4833c3c1.

**Comment by wking** [comment]
> PR needs rebase.

Rebased onto master with 4833c3c1 -> 8e15a8d8.  GitHub had complained about a `kubernetes-repositories.md` conflict, which might have been due to a9ef298b (#2277).  But rebasing resolved any conflicts without me having to do anything ;).  Let me know if there's anything I can do to get some or all of these landed; they seem straightforward enough that repeated rounds of rebasing and review are unnecessary time drains.

**Pull request closed without merging**